doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -698,6 +714,7 @@ It is proposed to handle these variances in the SAI implementation.
 
 ### 4.3.6 IMET route handling
```

Comment:
In case of P2MP tunnel there will be only one bridge port and thus there are some minor changes in code where instead of checking bridgeport alone fur duplicate, endpoint IP is also checked.